apps/dolly-frontend/src/main/js/src/components/fagsystem/sigrunstubSummertSkattegrunnlag/form/KjoeretoeyArrayForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { getInitialKjoeretoey } from '@/components/fagsystem/sigrunstubSummertSkattegrunnlag/form/Form'
+import { FormDollyFieldArray } from '@/components/ui/form/fieldArray/DollyFieldArray'
+import { FormSelect } from '@/components/ui/form/inputs/select/Select'
+import { FormTextInput } from '@/components/ui/form/inputs/textInput/TextInput'
+
+export const KjoeretoeyArrayForm = ({ path }) => (
+	<FormDollyFieldArray
+		name={path}
+		header="Spesifisering"
+		newEntry={getInitialKjoeretoey()}
+		canBeEmpty={true}
+		nested
+	>
+		{(kjoeretoeyPath, idx) => (
+			<div className="flexbox--flex-wrap sigrun-form" key={idx}>
+				<FormSelect
+					name={`${kjoeretoeyPath}.type`}
+					info={'Foreløpig er det kun støtte for kjøretøy'}
+					size={'large'}
+					label="Type"
+					value={'Kjoeretoey'}
+					isDisabled={true}
+					options={[
+						// { value: 'EiendelerOgFasteEiendommer', label: 'Eiendeler og faste eiendommer' },
+						{ value: 'Kjoeretoey', label: 'Kjøretøy' }, //Kun kjøretøy som er støttet foreløpig
+						// { value: 'Spesifisering', label: 'Spesifisering' },
+					]}
+				/>
+				<FormTextInput
```

Comment:
Skulle dette vært en dropdown? Evt. få på noe validering? For nå kan man skrive inn hva som helst 🤪